travis-ci/packer-templates
# Fixing the macos build script to run via packer

## What is the problem that this PR is trying to fix?
We used to build macos images manually and that made us sad.

## What approach did you choose and why?
Using packer! 

## How can you test this?
Locally running `make ci-macos BUILDER=vsphere`. Note that this does not test that the image can run customer builds successfully - this process just automates the creation of the image.

## What feedback would you like, if any?
